rmgpy/reactionTest.py

```diff
@@ -46,7 +46,9 @@
 from rmgpy.statmech.vibration import Vibration, HarmonicOscillator
 from rmgpy.statmech.torsion import Torsion, HinderedRotor
 from rmgpy.statmech.conformer import Conformer
-from rmgpy.kinetics import Arrhenius, ArrheniusEP, SurfaceArrhenius, StickingCoefficient
+from rmgpy.kinetics import Arrhenius, ArrheniusEP, MultiArrhenius, \
+                           PDepArrhenius, MultiPDepArrhenius, \
+                           SurfaceArrhenius, StickingCoefficient
```

Comment:
This can fit on two lines instead of three if you want. Is there a reason you went with three lines?

Reply:
I added line breaks by category, e.g. high-P, pdep, surface. I can put them on two lines if preferred.

Reply:
I see it now--this is fine then